assets/scss/components/stencil/cart/_cart.scss

```diff
@@ -541,12 +544,38 @@ $card-preview-zoom-bottom-offset:       6rem;
 
 .cart-additionalCheckoutButtons {
     @extend %additionalCheckoutButtons;
+    
+    @media (max-width: $screen-small) {
+        display: flex;
+        flex-direction: column;
+
+        >div {
```

Comment:
I believe > could be skipped here

Reply:
@BC-tymurbiedukhin I believe that we need to keep this type of styling (> div / > p) because we need to apply these styles only to the direct descendants of the "cart-additionalCheckoutButtons" container, namely only to "PayPalExpressCheckout" and "apple-pay-checkout-button" which can also contain inside their <div> and <p> tags, the styles of which it is desirable to keep intact.

<img width="1155" alt="465_tags_style" src="https://user-images.githubusercontent.com/82589781/121168589-cc4b5400-c85b-11eb-8021-558fba2bffed.png">

Reply:
In this case you should directly write classes for which properties should be applied